inputs-optional.tf

```diff
@@ -36,7 +36,7 @@ variable "ipconfig_name" {
 variable "privateip_allocation" {
   type        = string
   description = "The type of private IP allocation, either Static or Dynamic."
-  default     = "Static"
+  default     = "Dynamic"
```

Comment:
why setting to dynamic ?

Reply:
This was discussed with @timja. We both agree in that in most use cases the consumer of the module wouldn't be concerned with what IP was assigned so this should be the default but that it should be overridable if required.